### PR TITLE
aws 11.5.2

### PR DESCRIPTION
**Pull request by calvix**
for https://github.com/giantswarm/giantswarm/issues/12617

### CODE REVIEW

**Comment by calvix** [comment]
tested cluster creation of `v11.5.2` and update from `v11.5.1` to `v11.5.2` on gauss and ginger, everything looks good

File: aws/v11.5.2/release.diff [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    name: node-exporter                                                name: node-exporter
+    version: 1.2.0                                                     version: 1.2.0
+  components:                                                        components:
+  - name: app-operator                                               - name: app-operator
+    version: 1.0.0                                                     version: 1.0.0
+  - name: aws-cni                                                    - name: aws-cni
+    version: 1.6.0                                                     version: 1.6.0
+  - name: aws-operator                                               - name: aws-operator
+    releaseOperatorDeploy: true                                 |      version: 8.7.6
+    version: 8.7.5                                              <
```
Comment (xh3b4sd) [inline aws/v11.5.2/release.diff:47]: I am a bit confused what `|` and `<` means.

Comment (xh3b4sd) [reply aws/v11.5.2/release.diff:47]: The diff creates the impression that `releaseOperatorDeploy: true` is now not defined anymore, but in fact it is.

Comment (calvix) [reply aws/v11.5.2/release.diff:47]: this is generated by `devctl`, so I don't really know

Comment (xh3b4sd) [reply aws/v11.5.2/release.diff:47]: Maybe @tfussell can enlighten us. 😬

Comment (tfussell) [reply aws/v11.5.2/release.diff:47]: `|` means there's a difference on those two lines, `<` means the line is only on the left side. Please update `devctl` with [this PR](https://github.com/giantswarm/devctl/pull/87) and run this again to make sure you get `releaseOperatorDeploy: true` in your release.

File: aws/v11.5.2/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# :zap: Giant Swarm Release v11.5.2 for AWS :zap:
+
+This release provides a new aws-operator which is fixing an issue with NetworkPolicies and custom pod cidrs.
```
Comment (marians) [inline aws/v11.5.2/README.md:3]:
```suggestion
This release provides a new aws-operator which is fixing an issue with NetworkPolicies and custom Pod CIDRs.
```

File: aws/v11.5.2/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+## Change details
+
+### aws-operator [8.7.6](https://github.com/giantswarm/aws-operator/releases/tag/v8.7.6)
+- Add release version tag for ec2 instances.
+- Update Cloudformation Stack when components version differ.
+- Fix IAM policy on Tenant Clusters to manages IAM Role tags.
+- Fixed passing custom pod CIDR to k8scloudconfig.
```
Comment (marians) [inline aws/v11.5.2/README.md:11]:
```suggestion
- Fixed passing custom Pod CIDR to k8scloudconfig.
```

File: aws/v11.5.2/release.diff (hunk 1 above)
Comment (marians) [reply aws/v11.5.2/release.diff]: diff got updated

File: aws/v11.5.2/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# :zap: Giant Swarm Release v11.5.2 for AWS :zap:
+
+This release provides a new aws-operator which is fixing an issue with NetworkPolicies and custom Pod CIDRs.
+
+## Change details
+
+### aws-operator [8.7.6](https://github.com/giantswarm/aws-operator/releases/tag/v8.7.6)
```
Comment (njuettner) [inline aws/v11.5.2/README.md:7]:
```suggestion
### aws-operator [8.7.6](https://github.com/giantswarm/aws-operator/releases/tag/v8.7.6)

```

File: aws/v11.5.2/release.yaml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  - name: cluster-operator
+    releaseOperatorDeploy: true
+    version: 2.3.2
+  - name: containerlinux
+    version: 2512.2.1
+  - name: etcd
+    version: 3.4.9
+  - name: kubernetes
+    version: 1.16.13
+  date: "2020-08-14T10:16:08Z"
```
Comment (njuettner) [inline aws/v11.5.2/release.yaml:62]: Does the date have to stick with the day we want to release it?

Comment (marians) [reply aws/v11.5.2/release.yaml:62]: Yep

File: aws/v11.5.2/release.diff [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+  - name: cluster-operator                                           - name: cluster-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 2.3.2                                                     version: 2.3.2
+  - name: containerlinux                                             - name: containerlinux
+    version: 2512.2.1                                                  version: 2512.2.1
+  - name: etcd                                                       - name: etcd
+    version: 3.4.9                                                     version: 3.4.9
+  - name: kubernetes                                                 - name: kubernetes
+    version: 1.16.13                                                   version: 1.16.13
+  date: "2020-08-04T12:34:57Z"                                  |    date: "2020-08-14T10:16:08Z"
```
Comment (marians) [inline aws/v11.5.2/release.diff:62]:
```suggestion
  date: "2020-08-04T12:34:57Z"                                  |    date: "2020-08-17T12:00:00Z"
```

File: aws/v11.5.2/release.yaml (hunk 5 above)
Comment (marians) [inline aws/v11.5.2/release.yaml:62]:
```suggestion
  date: "2020-08-17T12:00:00Z"
```

Comment (njuettner) [reply aws/v11.5.2/release.yaml:62]: Thanks 🙂, I was not sure about that

**Pull request merged**